pkg/detector/detector.go

```diff
@@ -1171,28 +1168,22 @@ func (d *ResourceDetector) HandleClusterPropagationPolicyDeletion(policyName str
 		policyv1alpha1.ClusterPropagationPolicyLabel: policyName,
 	}
 
+	removeLabels := func(labels map[string]string) map[string]string {
```

Comment:
minor flaw, I think this variable name `removeLabels` is a little simple and violent, it could be improved

Reply:
Any suggestion?

Reply:
As for me, I would prefer declare a common function like:

```go
func TrimMapByKeys(inputMap map[string]interface{}, keys []string) map[string]interface{}
```

This name clearly conveys the purpose and action of the function. It indicates that the function trims/removes a slice of specific key-value pair from the input map and returns the updated map as the output. 

By including "Trim" and "ByKeys" in the function name, it provides a clear indication of the operation being performed on the map.

Then, we invoke the function as `TrimMapByKeys(workload.GetLabels(), propagationPolicyRefLabels)`

In this case, may be this function will not be required as a parameter and been passed on for many times.

However, just for reference only.

Reply:
The idea here is not to bring the specific label key to the bottom layer for cleaning. For the bottom layer, it only needs to return the processed label. Therefore, an anonymous function is used here for processing.